src/test/fuzz/validation_load_mempool.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <chainparamsbase.h>
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+#include <test/util/setup_common.h>
+#include <txmempool.h>
+#include <validation.h>
+
+#include <cstdint>
+#include <vector>
+
+namespace {
+FuzzedFileProvider* fuzzed_file_provider_ptr = nullptr;
+
+FILE* fuzzed_fopen(const fs::path&, const char*)
+{
+    return fuzzed_file_provider_ptr->open();
+}
+} // namespace
+
+void initialize()
+{
+    static TestingSetup setup{CBaseChainParams::REGTEST, {"-nodebuglogfile"}};
+}
+
+void test_one_input(const std::vector<uint8_t>& buffer)
+{
+    FuzzedDataProvider fuzzed_data_provider{buffer.data(), buffer.size()};
+    FuzzedFileProvider fuzzed_file_provider = ConsumeFile(fuzzed_data_provider);
+    fuzzed_file_provider_ptr = &fuzzed_file_provider;
+
+    CTxMemPool pool;
+    (void)LoadMempool(pool, fuzzed_fopen);
```

Comment:
Time could be mocked since in `LoadMempool` there is a call to `GetTime`.

Reply:
Good point! Now fixed. Please re-review :)